### PR TITLE
chore: release 0.29.0

### PR DESCRIPTION
**Pull request by thbcmlowk**
Closes FL-1031

### CODE REVIEW

File: docs/source/additional/releases.yaml [hunk 1]
```diff
@@ -9,6 +9,44 @@ components: # for table headers
...
+          link: https://artifacthub.io/packages/helm/substra/substra-frontend/1.0.20
+      hlf-k8s:
+        version: 0.2.4
+        link: https://github.com/Substra/hlf-k8s/releases/tag/0.2.4
+        helm:
+          version: 10.2.4
+          link: https://artifacthub.io/packages/helm/substra/hlf-k8s/10.2.4
+      substra-tests:
+        version: 0.41.0
+        link: https://github.com/Substra/substra-tests/releases/tag/0.41.0
```
[the remark this answers is not in the document]
Comment (thbcmlowk) [reply docs/source/additional/releases.yaml:49]: I'm confused. `substra-tests` hasn't been released, has it?

[the remark this answers is not in the document]
Comment (ThibaultFy) [reply docs/source/additional/releases.yaml:49]: It should be release here https://github.com/Substra/substra-tests/pull/262

[the remark this answers is not in the document]
Comment (thbcmlowk) [reply docs/source/additional/releases.yaml]: Yes indeed, nicely spotted! Thanks

**Review by ThibaultFy — APPROVED** [review]
Thanks for the release 🚀

**Pull request merged**